x-pack/plugins/apm/public/components/app/RumDashboard/RumHome.tsx

```diff
@@ -20,24 +20,15 @@ export const UX_LABEL = i18n.translate('xpack.apm.ux.title', {
 export function RumHome() {
   return (
     <CsmSharedContextProvider>
-      <EuiFlexGroup>
+      <EuiFlexGroup wrap justifyContent={'flexEnd'} responsive={true}>
```

Comment:
Nit: this could be `justifyContent="flexEnd"

Reply:
i have no idea, why our lint doesnt enforce this rule